src/OrchardCore.Modules/OrchardCore.Roles/Migrations/RolesMigrations.cs

```diff
@@ -41,10 +46,8 @@ public int Create()
                     continue;
                 }
 
-                // When a new tenant is created, the RoleClaims will be empty for Admin roles.
```

Comment:
Do you remember what this comment was about? This is not hit for new tenants, isn't it?

Reply:
correct. this is only when migrating sites from previous version.